umm-spec-lib/src/cmr/umm_spec/iso19115_2_util.clj

```diff
@@ -13,6 +13,14 @@
 (def short-name-xpath
   "gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString")
 
+;; relative to gmi:childOperation
+(def campaign-xpath
+  "gmi:MI_Operation/gmi:identifier/gmd:MD_Identifier/gmd:code/gco:CharacterString") 
+
+(def start-end-date-xpath
```

Comment:
This name here is confusing because it's in the iso utils and called start-end-date, but description has nothing to do necessarily with start-end-date and "gmi:description/gco:CharacterString" is widely used throughout iso.